defect-tracker-server/src/main/java/com/defect/tracker/util/Constants.java

```diff
@@ -1,11 +1,13 @@
 package com.defect.tracker.util;
 
+
 public class Constants {
 
 	// project
 	public static final String PROJECT_ADDED_SUCCESS = "Project added successfully ";
 	public static final String DESIGNATION_ADDED_SUCCESS = "Designation added successfully";
 	// Encapsulate constructor to restrict modification from outside
+	public static final String DELETED_SUCCESS = "Delete successfully";
```

Comment:
It is better to do so --->DESIGNATION_DELETED_SUCCESS = "Designation  delete successfully";